vagrant-pxe-harvester/ansible/roles/https/handlers/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: restart nginx for https
```

Comment:
Since it's doing the same thing as 'restart nginx', maybe better to put it in a separate role so it can be shared between http and https. See https://stackoverflow.com/questions/38188717/how-to-share-handlers

Reply:
it seems we can use dependencies for the case as the https://stackoverflow.com/a/29297938 suggested.

tested on case `http:true`
![image](https://user-images.githubusercontent.com/5169694/145251546-c516946d-7ef5-41bc-9da5-90c89f86eaba.png)

and case `http:false`
![image](https://user-images.githubusercontent.com/5169694/145251693-65d17081-190b-483b-b043-ffb603faf892.png)